go/teams/loader2.go

```diff
@@ -163,13 +163,13 @@ func (l *TeamLoader) checkStubbed(ctx context.Context, arg load2ArgT, link *chai
 		return nil
 	}
 	if l.seqnosContains(arg.needSeqnos, link.Seqno()) {
-		return NewStubbedErrorWithNote(link, "need seqno")
+		return NewStubbedErrorWithNote(link, "Need seqno")
 	}
 	if arg.needAdmin {
-		return NewStubbedErrorWithNote(link, "need admin")
+		return NewStubbedErrorWithNote(link, "Need admin privilege for this action")
 	}
 	if !link.outerLink.LinkType.TeamAllowStubWithAdminFlag(arg.needAdmin) {
-		return NewStubbedErrorWithNote(link, "disallowed")
+		return NewStubbedErrorWithNote(link, "This action is disallowed")
```

Comment:
I don't think this note is accurate. This case is when a link was stubbed but we wanted to load as an admin. Assuming the system is functioning, this case will is the same as the previous. So might as well give it the same note.